bp/parser.js

```diff
@@ -19,33 +19,28 @@ module.exports = new Parser(function analyseEC(parsedUrl, ec) {
   // use console.error for debuging
   // console.error(parsedUrl);
 
-  let match;
-
-  if ((match = /^\/monographs\/([0-9a-z-]+)\.pdf$/i.exec(path)) !== null) {
+  if (/^\/monographs\/([0-9a-z-]+)\.pdf$/i.test(path)) {
```

Comment:
Why did you remove the unitit. it is correct before. With your version, you will not be able to deduplicate consultations of this parser